test/server/contentBase-option.test.js

```diff
@@ -301,6 +301,29 @@ describe('contentBase option', () => {
         done();
       }
     });
+    it('Should not throw exception (local path with lower case first character)', (done) => {
+      try {
+        // eslint-disable-next-line no-unused-vars
+        server = new Promise((resolve, reject) => {
+          testServer.start(config, {
+            contentBase:
```

Comment:
Let's add two tests here `c:\absolute\path\to\content-base` and `C:\absolute\path\to\content-base`